src/main/scala/org/ergoplatform/nodeView/history/storage/HistoryStorage.scala

```diff
@@ -32,7 +32,7 @@ class HistoryStorage(indexStore: LDBKVStore, objectsStore: LDBKVStore, config: C
     .build[ByteArrayWrapper, Array[Byte]]
 
   def modifierBytesById(id: ModifierId): Option[Array[Byte]] =
-    objectsStore.get(idToBytes(id))
+    objectsStore.get(idToBytes(id)).map(_.tail)
```

Comment:
@kushti please leave a comment. What this means?

Reply:
@aslesarenko fix for https://github.com/ergoplatform/ergo/issues/1547 , will add in 4.0.20